spec/controllers/dashboards_controller_spec.rb

```diff
@@ -148,18 +148,14 @@ def load_table(klass, options)
       end
 
       context 'when successful' do
-        it 'adds a new version record' do
-          SiteMapperHelper.silence do
-            expect { post(:push) }.to change(Version, :count).by(1)
-          end
+        before do
```

Comment:
Empty `before` block?

Reply:
Sorry, left over code from when I was trying to fix broken tests. Changes are pushed in.